index.html

```diff
@@ -3,7 +3,7 @@
         <title>My website</title>
```

Comment:
As for the switch to PHP, can you see if you can commit your updated code and sync it with Github. After doing that, you can check that it shows up in this Pull Request. Let me know when it's updated, or if you're still having problems.